lenovolegionlinux-dkms/PKGBUILD

```
@@ -0,0 +1,35 @@
+# Maintainer: MrDuartePT <gonegrier.duarte@gmail.com>
+# Maintainer: xenhat <aur@xenh.at>
+# Maintainer: johnfanv2 <https://github.com/johnfanv2>
+
+_pkgname=lenovolegionlinux
+pkgname=${_pkgname}-dkms
+pkgver=0.0.18
+pkgrel=0
+pkgdesc="LenovoLegionLinux (LLL) DKMS module"
+arch=("x86_64")
+url="https://github.com/johnfanv2/LenovoLegionLinux"
+license=('GPL')
+makedepends=(
+	"lm_sensors"
+	"i2c-tools" 
+	"dmidecode"
+	"lenovolegionlinux-git"
+)
+depends=(
+	"dkms"
+    "lenovolegionlinux-git"
```

```suggestion
    "lenovolegionlinux"
```
Should be this right?

---

Yes, forget to change that